test/performance_tests.jl

```diff
@@ -6,6 +6,7 @@ import CloudMicrophysics as CM
 import ClimaParams as CP
 import Thermodynamics as TD
 
```

Comment:
Thank you for splitting that part of your larger PR. Would it be possible to also add some sort of unit test for this, and not only performance tests?